el/django_templates/README.md

```diff
@@ -0,0 +1,106 @@
+# Django templates
+
+Ήρθε η ώρα να παρουσιάσουμε ορισμένα δεδομένα! Το Django μας δίνει κάποια προεγκατεστημένα **template tags**.
+
+## Τι είναι τα template tags;
+
+Βλέπετε, στην HTML δεν μπορείτε να γράψετε κώδικα σε Python επειδή οι browsers δεν καταλαβαίνουν αυτή τη γλώσσα. Εκείνοι ξέρουν μόνο HTML, CSS και Javascript. Ξέρουμε ότι η HTML είναι στατική γλώσσα ενώ η Python περισσότερο δυναμική.
+
+Τα **Django template tags** μας επιτρέπουν να μεταφέρουμε πράγματα της Python μέσα στην HTML, ούτως ώστε να χτίσετε δυναμικές ιστοσελίδες γρηγορότερα. Τέλεια!
+
+## Template προβολή λίστας των post
+
+Σε προηγούμενο κεφάλαιο δώσαμε στο template μας μια λίστα από posts μέσα σε μια μεταβλητή με το όνομα `posts`. Τώρα θα την προβάλουμε στην HTML. Άλλωστε αυτό θέλαμε να κάνουμε από την αρχή.
+
+Για να εκτυπώσουμε μια μεταβλητή μέσα στα Django templates, χρησιμοποιούμε διπλές αγκύλες ({{ }}) με το όνομα της μεταβλητής ανάμεσα:
```

Comment:
@manikos what is this `({{ }})` here for?

Reply:
In Greek language we describe `[`, `]`, `{` and `}` as one word (αγκύλη == brace). So, in order not to confuse them I wrote it inside a parenthesis. As I can see the build fails. Should I remove it?

Reply:
`{% raw %}` most probably should do the trick here. Generally it should not fail. You can also try to build it locally (using node and gitbook).

And, please, apply change on Crowdin (so I will be able to rebuild this pull request).

Reply:
I'll do this right now.

Reply:
Great!

Reply:
Done!